frontends/main/src/app-pages/HomePage/BrowseTopicsSection.tsx

```diff
@@ -119,6 +121,11 @@ const BrowseTopicsSection: React.FC = () => {
                 <TopicBox
                   key={id}
                   href={channelUrl ? new URL(channelUrl!).pathname : ""}
+                  onClick={() => {
+                    if (process.env.NEXT_PUBLIC_POSTHOG_API_KEY) {
+                      posthog.capture("home_topic_clicked", { topic: name })
```

Comment:
something to consider - we have a bunch of these bare event strings. it might be good to have them defined as constants somewhere so its easier to track and change